deploy/README.md

```diff
@@ -1,4 +1,4 @@
-# Deployment using Operators
+# SAF Deployment using Operators
```

Comment:
Should probably just call this out in the first instance:  Service Assurance Framework

Reply:
Of course I'm not blocking this for this change :)

Reply:
Fixed